src/main/java/databaseInteracting/Client.java

```diff
@@ -0,0 +1,163 @@
+        System.out.println(response);
+      }
+      if (option == 3) {
+        sc.nextLine();
+        System.out.println("Type the product name");
+        String name = sc.nextLine();
+        System.out.println("Type the number of product inserted on stock");
+        int stock = sc.nextInt();
+        System.out.println("Type the price of the product");
+        float price = sc.nextFloat();
+        addProductRequest request =
+            addProductRequest.newBuilder().setName(name).setStock(stock).setPrice(price).build();
+        addProductResponse response = prodStub.addProduct(request);
+        System.out.println(response.getName());
+      }
+      if (option == 4) {
+        sc.nextLine();
+        System.out.println("Type the client name");
+        String name = sc.nextLine();
+        createAShoppingCartRequest createAShoppingCartRequest =
+            generated.createAShoppingCartRequest.newBuilder().setName(name).build();
+        createAShoppingCartResponse response = prodStub.createAShoppingCart(createAShoppingCartRequest);
+        System.out.println("Cart id: " + response.getId() + "Owner Name: " + response.getName());
+      }
+
+      if (option == 5) {
+        System.out.println("Type the id of shopping cart");
+        int idCart = sc.nextInt();
+        System.out.println("Type the id of desired product:");
+        int idProduct = sc.nextInt();
+        System.out.println("Type the quantity of this product:");
+        int quantity = sc.nextInt();
+        addProductsToShoppingCartRequest request =
+            addProductsToShoppingCartRequest.newBuilder().setIdShoppingCart(idCart).setIdProduct(idProduct)
+                .setQuantity(quantity).build();
+        addProductsToShoppingCartResponse response = prodStub.addProductsToShoppingCart(request);
+        System.out.println("Shopping Cart Id: " + response.getIdShoppingCart() + " Product ID: "
+            + response.getIdProduct() + " Quantity: " + response.getQuantity());
+      }
+      if (option == 6) {
+        System.out.println("Type your Shopping Cart ID: ");
+        int id = sc.nextInt();
+        System.out.println("Your Shopping Cart has the products below:");
+         try {
+          listShoppingCartProductsRequest request =
+              listShoppingCartProductsRequest.newBuilder().setCartId(id).build();
+          Iterator<listShoppingCartProductsResponse> listShoppingCartProductsResponseIterator =
+              prodStub.listShoppingCartProducts(request);
+          while (listShoppingCartProductsResponseIterator.hasNext()) {
+            listShoppingCartProductsResponse next = listShoppingCartProductsResponseIterator.next();
+            System.out.printf("Product Id: %d Product Name: %s  Price: %.2f Quantity: %d \n",
+                next.getIdProduct(), next.getName(), next.getPrice(), next.getQuantity());
+            }
+          } catch (Exception e) {
+          e.printStackTrace();
```

Comment:
Not a good practice to print the stack trace

Reply:
done.